values.yaml

```diff
@@ -401,13 +390,19 @@ container_images:
         registry: replicas.mirror.gpkg.io/proxy-docker-io
         repository: curlimages/curl
         tag: 8.16.0@sha256:463eaf6072688fe96ac64fa623fe73e1dbe25d8ad6c34404a669ad3ce1f104b6
+  app_gluekube_ccm:
+    gluekube_ccm:
+      image:
+        registry: replicas.mirror.gpkg.io/proxy-quay-io
+        repository: glueops/ccm
+        tag: v51
```

Comment:
can you tag with semver vMAJOR.MINOR.PATCH

so: v52.0.0 (since v51 already exists)